dtcli/building.py

```diff
@@ -43,7 +43,7 @@ def _zip_extension(extension_dir_path, extension_zip_path):
     utils.check_file_exists(extension_zip_path)
     print("Building %s from %s" % (extension_zip_path, extension_dir_path))
 
-    with zipfile.ZipFile(extension_zip_path, "w") as zf:
+    with zipfile.ZipFile(extension_zip_path, "w", zipfile.ZIP_DEFLATED) as zf:
```

Comment:
Why DEFLATED and not one of the other 2 possible? What's the `compresslevel` with default=None? Why not 9? Was that tested against a dev tenant?